src/modules/interface/msg/Tf_rotate.msg

```diff
@@ -1,3 +1,7 @@
 Header header
 
-geometry_msgs/TwistStamped imuStamp        #Contains rotation from IMU
\ No newline at end of file
+# Contains rotation from IMU, where
+# x = roll
+# y = pitch
+# z = yaw
+geometry_msgs/TwistStamped imuStamp
```

Comment:
Why do we use twist? Twist is for linear and angular velocity.

Reply:
I recommend the following messages:
TwistWithCovarianceStamped
AccelWithCovarianceStamped
PoseWithCovarianceStamped

and any other suggestions you may have, please document them here.